pages/login.js

```diff
@@ -0,0 +1,61 @@
+import Head from 'next/head';
+import { useState } from 'react';
+import styles from '../styles/Home.module.css';
+
+export default function Login() {
+  const [error, setError] = useState('');
+
+  function onSubmit(event) {
+    event.preventDefault()
+
+    // Get form data and create JSON object
+    const formData = new FormData(event.target)
+    const formDataObj = {}
+    formData.forEach((value, key) => (formDataObj[key] = value))
+
+    let username = formDataObj["username"]
+    let password = formDataObj["password"]
```

Comment:
Why are you putting `formData` into a separate object? Can't we just iterate over `formData` instead?